app/src/features/projects/view/ProjectPage.test.tsx

```diff
@@ -78,19 +80,43 @@ describe('ProjectPage', () => {
     });
   });
 
-  it('delete project works and takes user to the projects list page', async () => {
+  it('deletes project works and takes user to the projects list page when user is a system administrator', async () => {
     mockBiohubApi().codes.getAllCodeSets.mockResolvedValue({
       activity: [{ id: 1, name: 'activity 1' }]
     } as any);
     mockBiohubApi().project.getProjectForView.mockResolvedValue(getProjectForViewResponse);
     mockBiohubApi().project.deleteProject.mockResolvedValue(true);
 
+    const hasSystemRoleMock = jest.fn().mockReturnValueOnce(true).mockReturnValueOnce(false);
+
+    const authState = {
+      keycloakWrapper: {
+        keycloak: {
+          authenticated: true
+        },
+        hasLoadedAllUserInfo: true,
+        systemRoles: [SYSTEM_ROLE.SYSTEM_ADMIN] as string[],
+        getUserIdentifier: () => 'testuser',
+        hasAccessRequest: false,
+        hasSystemRole: hasSystemRoleMock,
+        getIdentitySource: () => 'idir',
+        username: 'testusername',
+        displayName: 'testdisplayname',
+        email: 'test@email.com',
+        firstName: 'testfirst',
+        lastName: 'testlast',
+        refresh: () => {}
+      }
+    };
```

Comment:
can we define this once in the top describe block and just use `{ ...authState, systemRoles: blah }` as necessary?
makes it easier to read/understand what is actually changing and also smaller code